### ISSUE TITLE
Format for regional locales is inconsistent

### ISSUE THREAD

**Issue by jbphet** [phetsims/babel#25]
In https://github.com/phetsims/weddell/issues/134, it was reported that the format of the regional locales is inconsistent.  It's true - somethings a comma is used and sometimes parentheses are used.  Here's a short excerpt of `babel\localeData.json` that illustrates this:

```json
  "ar_LY": {
    "englishName": "Arabic (Libya)",
    "localizedName": "العربية (ليبيا)",
    "direction": "rtl",
    "fallbackLocales": [
      "ar",
      "ar_MA"
    ]
  },
  "ar_MA": {
    "locale3": "ary",
    "englishName": "Arabic, Morocco",
    "localizedName": "العربية (المغرب)",
    "direction": "rtl",
    "fallbackLocales": [
      "ar"
    ]
  },
```

I'm not sure how this came about.  It may have been like this for years.

**Comment by jbphet** [comment]
In addition to affecting the desktop app, it also impacts the translation utility.  Here's a screenshot from the "Select Local" dropdown menu:

![image](https://github.com/user-attachments/assets/71d23d2d-a11b-4677-b94e-989789679cae)

**Comment by jbphet** [comment]
@mattpen and @jonathanolson, I'd like your input on the following question:  The `localizedName` field seems to consistently use the format "locale (region)", whereas the `englishName` field does not - it sometimes uses the format "locale, region".  Would you be okay if I just converted all the English names to match?

**Comment by mattpen** [comment]
I think it should all be "locale (region)" - yes that sounds great!

**Comment by mattpen** [comment]
Please assign @chrisklus and @mattpen when complete so we know to import this to the website.

**Comment by jonathanolson** [comment]
@solaolateju thoughts on this formatting for regional locales?

**Comment by solaolateju** [comment]
I agree with @mattpen. Parentheses are most commonly used -  Locale(region). 
That is true with other popular websites.
<img width="319" alt="image" src="https://github.com/user-attachments/assets/ccdbda0c-a52d-4ae4-b3f6-f338f9551c0d">
<img width="175" alt="image" src="https://github.com/user-attachments/assets/b26350e5-2c95-48fd-a609-46de909cc73e">
